docs/admin/deployment.md

```diff
@@ -2,6 +2,115 @@
 
 ### Heroku
 
+#### Account setup on Heroku
+Setting up a simple Heroku “Free Account” is easy and should take no more than a couple of minutes. For this guide I will be using a fictitious email account “postgrest@kismail.ru” to set up the account on Heroku. You can use whatever email account you desire.
+NOTE: During signup you will be asked to “Pick your primary development language”. It is safe here to choose any language or simply “I use another language”.
+
+1. Heroku signup input
```

Comment:
You can avoid the `<br>`s here and use the list behavior I think. Something like

```markdown
1. Heroku signup input ![heroku signup input](https://github.com/Mithrandir21/postgrest/blob/master/docs/img/heroku_guide/1_heroku_signup_input.png)
2. Heroku signup waiting ![heroku signup waiting](https://github.com/Mithrandir21/postgrest/blob/master/docs/img/heroku_guide/2_heroku_signup_waiting.png)
3. Heroku signup confirmation ![heroku signup confirmation](https://github.com/Mithrandir21/postgrest/blob/master/docs/img/heroku_guide/3_heroku_signup_confirmation.png)
4. Heroku signup password ![heroku signup password](https://github.com/Mithrandir21/postgrest/blob/master/docs/img/heroku_guide/4_heroku_signup_password.png)
```

Also these absolute path image links should be replaced by relative links such as `img/heroku_guide/1_heroku_signup_input.png`.